### ISSUE TITLE
Database permissions section missing

### ISSUE THREAD

**Issue by LoneMarauder** [MicrosoftDocs/windowsserverdocs#5177]
This section of the document starts off with applying permissions to the database, but then switches back to adding a login to the SQL server - specifically, adding a machine login.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 29f08e7d-a6a0-23f3-f00e-bc8a1c8c5e2e
* Version Independent ID: 6b1b1c98-0a0a-6e41-558b-5cd85c33934d
* Content: [Migrating the WSUS Database from (Windows Internal Database) WID to SQL](https://docs.microsoft.com/en-us/windows-server/administration/windows-server-update-services/manage/wid-to-sql-migration)
* Content Source: [WindowsServerDocs/administration/windows-server-update-services/manage/wid-to-sql-migration.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/administration/windows-server-update-services/manage/wid-to-sql-migration.md)
* Product: **windows-server**
* Technology: **wsus**
* GitHub Login: @eross-msft
* Microsoft Alias: **lizross**

**Comment by illfated** [comment]
Well, there is a part about Database Permissions further down, it looks pretty OK to me (even though I have not used WSUS on the production server at my workplace).
(https://docs.microsoft.com/en-us/windows-server/administration/windows-server-update-services/manage/wid-to-sql-migration#database-permissions)
I guess I would have to test this on a reasonable test lab computer to see if it works as documented.

**Comment by LoneMarauder** [comment]
No, the section labeled "Database Permissions" is exactly the same as the "SQL Server Login Permissions" section - the only difference is that the "Database Permissions" section shows how to add a machine account, where the "SQL Server Login Permissions" section shows how to add the "NT AUTHORITY\NETWORK SERVICE" account.

**Comment by illfated** [comment]
Well, I guess I don't know enough about this topic to say what is correct and what is not correct.

Do you have a suggestion for how it needs to be written instead?

**Comment by IngridAtMicrosoft** [comment]
#assign:IngridAtMicrosoft 
gh issue comment 5174 -b #assign:IngridAtMicrosoft

**Comment by robinharwood** [comment]
We hugely appreciate your contribution, feedback, and support to our technical documentation.

We’ve phased out GitHub Issues on this repository as the feedback mechanism for content and have replaced it with a new feedback system. For more information see, [Content User Feedback](https://aka.ms/ContentUserFeedback). As part of this rollout, we are migrating your issue where our team will use your comments to improve the documentation experience. Unfortunately, this does mean you won't be able to access this issue after the migration is complete.

All issues will be migrated by Thursday 29th August 2024.

Once again, thank you for your valuable feedback, we hope you’ll continue to contribute using our new feedback system.
